src/Markdig.Wpf/MarkdownViewer.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Documents;
+
+namespace Markdig.Wpf
+{
+    /// <summary>
+    /// A markdown viewer control.
+    /// </summary>
+    public class MarkdownViewer : Control
```

Comment:
Yep, that's definitely something we should add! I was thinking of adding it before submitting, bit I wanted to get a MVP in there first ;)